### PR TITLE
Ajouter les champs de dénomination usuelle à la constitution du nom complet de l'unité légale

### LINKED ISSUE #110: Unifier les champs utilisés pour la dénomination complète de l'unité légale

### PR DESCRIPTION
**Pull request by HAEKADI**
closes #110

### CODE REVIEW

**Review by XavierJp — APPROVED** [review]
N'avait-on pas parlé également de prenomUsuel a la place de prenom1 ?

File: data_enrichment.py [hunk 1]
```diff
@@ -45,6 +48,14 @@ def format_nom_complet(
         name = f"{prenom if prenom else ''}{formatted_name}"
     if nom_raison_sociale:
         name = nom_raison_sociale
+    if denomination_usuelle_1 or denomination_usuelle_2 or denomination_usuelle_3:
```
Comment (geoffreyaldebert) [inline data_enrichment.py:51]: not sure about this piece of code. If one of `denomination_usuelle` is `None` it won't work.

File: data_enrichment.py [hunk 2]
```diff
@@ -45,6 +48,14 @@ def format_nom_complet(
...
+        all_denomination_usuelle = (
+            f"{denomination_usuelle_1}"
+            f" {denomination_usuelle_2} "
+            f"{denomination_usuelle_3}"
+        )
+        if all_denomination_usuelle:
```
Comment (geoffreyaldebert) [inline data_enrichment.py:57]: same here. It always exists as you declare just above and you already conditionned it.

File: data_enrichment.py [hunk 3]
```diff
@@ -45,6 +48,14 @@ def format_nom_complet(
         name = f"{prenom if prenom else ''}{formatted_name}"
     if nom_raison_sociale:
         name = nom_raison_sociale
+    if denomination_usuelle_1 or denomination_usuelle_2 or denomination_usuelle_3:
+        all_denomination_usuelle = (
```
Comment (geoffreyaldebert) [inline data_enrichment.py:52]: You can have a problem of double space in the middle here.
I propose : 
``` 
all_denomination_usuelle = ""
for item in [denomination_usuelle_1, denomination_usuelle_2, denomination_usuelle_3]:
    if item:
        all_denomination_usuelle += f"{item} "
 ```

**Review by geoffreyaldebert — CHANGES_REQUESTED** [review]

File: data_enrichment.py (hunk 1 above)
Comment (HAEKADI) [reply data_enrichment.py:51]: It's an `or` condition  so it works if at least one is not `None`.

Comment (HAEKADI) [reply data_enrichment.py:51]: Yes. I noticed it, but I thought it was highly unlikely to have `denomination_3` and not `denomination_2`, so I chose to ignore it.
But you're right, I should also prepare for this possibility.

**Pull request closed without merging**